schema/url/differentialabundance/plugin-differentialabundance-volcanoplot.raml

```diff
@@ -0,0 +1,17 @@
+#%RAML 1.0 Library
+
+types:
+  # Type name determines the name of the Java class that will be generated.
+
+  DifferentialAbundanceVolcanoplotPostRequest:
+    type: DataPluginRequestBase
+    properties:
+      computeConfig: DifferentialAbundanceComputeConfig
+      config: DifferentialAbundanceVolcanoplotSpec
+
+  # No variables, no missingness
+  DifferentialAbundanceVolcanoplotSpec:
+    additionalProperties: false
+    properties:
+      outputEntityId:
```

Comment:
lol. they cant exactly change the output entity, since this plot doesnt belong in an entity space that actually exists in the entity diagram. is it possible this has no viz config?